bevy_rapier_benches3d/Cargo.toml

```diff
@@ -8,7 +8,20 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+visual = [
+    "bevy/x11",
+    "bevy/tonemapping_luts",
+    "bevy/bevy_asset",
+    "bevy/bevy_scene",
+    "bevy/bevy_core_pipeline",
+    "bevy/bevy_pbr",
+    "bevy/bevy_gizmos",
+    "rapier3d/debug-render",
+    "bevy/bevy_asset",
+]
+
 [dependencies]
-rapier3d = { features = ["profiler"], version = "0.22" }
-bevy_rapier3d = { version = "0.27", path = "../bevy_rapier3d" }
 bevy = { version = "0.14", default-features = false }
+rapier3d = { features = ["profiler"], version = "0.22" }
```

Comment:
This makes me think that I’d like to revisit that `profiler` thing in `rapier` to switch so something less manual. Like switching the `tracy` or something. (This is unrelated to this PR though.)